tests/integration/integration_test.go

```diff
@@ -940,6 +967,136 @@ status: {}
+kind: ProfileInstallation
+metadata:
+  creationTimestamp: null
+  name: pctl-profile
+  namespace: %s
+spec:
+  source:
+    branch: dependson
+    path: dependson-nginx
+    url: https://github.com/weaveworks/profiles-examples
+status: {}
+`, namespace)))
+
+				By("verify that dependsOn has been added to the helm release")
+				filename = filepath.Join(temp, "artifacts", "nginx-chart", "HelmRelease.yaml")
+				content, err = ioutil.ReadFile(filename)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(string(content)).To(Equal(fmt.Sprintf(`apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  creationTimestamp: null
+  name: pctl-profile-dependson-nginx-nginx-chart
+  namespace: %s
+spec:
+  chart:
+    spec:
+      chart: nginx
+      sourceRef:
+        kind: HelmRepository
+        name: pctl-profile-profiles-examples-nginx
+        namespace: %s
+      version: 9.3.0
+  dependsOn:
+  - name: pctl-profile-dependson-nginx-dependon
+    namespace: %s
+  interval: 0s
+  valuesFrom:
+  - kind: ConfigMap
+    name: pctl-profile-nginx-chart-defaultvalues
+    valuesKey: default-values.yaml
+status: {}
+`, namespace, namespace, namespace)))
+
+				By("the artifacts being deployable")
+
+				cmd = exec.Command("kubectl", "apply", "-R", "-f", profilesDir)
+				cmd.Dir = temp
+				output, err = cmd.CombinedOutput()
+				Expect(err).ToNot(HaveOccurred(), fmt.Sprintf("kubectl apply failed : %s", string(output)))
+
+				By("successfully deploying the chart resource")
+				helmReleaseName := fmt.Sprintf("%s-%s-%s", subName, "dependson-nginx", "nginx-chart")
+				var helmRelease *helmv2.HelmRelease
+				Eventually(func() bool {
+					helmRelease = &helmv2.HelmRelease{}
+					err := kClient.Get(context.Background(), client.ObjectKey{Name: helmReleaseName, Namespace: namespace}, helmRelease)
+					if err != nil {
+						return false
+					}
+					for _, condition := range helmRelease.Status.Conditions {
+						if condition.Type == "Ready" && condition.Status == "True" {
+							return true
+						}
+					}
+					return false
+				}, 5*time.Minute, 5*time.Second).Should(BeTrue())
+				// it's enough to verify the nginx installation because it wouldn't start up if redis wouldn't be running.
+				helmReleaseOpts := []client.ListOption{
+					client.InNamespace(namespace),
+					client.MatchingLabels{"helm.sh/chart": "nginx-9.3.0"},
+				}
+				podList := &v1.PodList{}
+				Eventually(func() v1.PodPhase {
+					podList = &v1.PodList{}
+					err := kClient.List(context.Background(), podList, helmReleaseOpts...)
+					Expect(err).NotTo(HaveOccurred())
+					if len(podList.Items) == 0 {
+						return v1.PodPhase("no pods found")
+					}
+					return podList.Items[0].Status.Phase
+				}, 2*time.Minute, 5*time.Second).Should(Equal(v1.PodPhase("Running")))
+				Expect(podList.Items[0].Spec.Containers[0].Image).To(Equal("docker.io/bitnami/nginx:1.21.0-debian-10-r0"))
+			})
+		})
```

Comment:
rather than a new test can we just modify the existing test? This files a 1000 lines :sob:

Reply:
Hmm, hmmm maybe. :D

Reply:
If I modify the existing profile like `weaveworks-nginx`, it will trigger update to almost ever test as in add to the output new information and whatnot. The net change would be the same number of lines as with a new test, but would be more confusing.